guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_prerequisites.adoc

```diff
@@ -19,6 +19,10 @@ Cloning is the recommended method because that prevents them being overwritten i
 To confirm if a template has been edited, you can view its *History* before you upgrade or view the changes in the audit log after an upgrade.
 In the {ProjectWebUI}, navigate to *Monitor* > *Audits* and search for the template to see a record of changes made.
 If you use the export method, restore your changes by comparing the exported template and the default template, manually applying your changes.
+ifdef::satellite[]
+* If you previously upgraded {Project} from an earlier version, and the `/var/lib/pgsql` directory contains any content, empty `/var/lib/pgsql` before proceeding.
+* {Project} {ProjectVersionPrevious}'s migrated `postgres` database content is stored in this directory.
```

Comment:
This reads as if `/var/lib/pgsql` contains the *current* database content, which it does not. It does contain some old, not-cleaned-up content. I'd probably either completely remove this sentence, or reword it to something like: `/var/lib/pgsql` contained the PostgreSQL database content before the migration from PostgreSQL 9 to PostgreSQL 12 from the SCL.